packages/fields/tsconfig.json

```diff
@@ -23,7 +23,9 @@
 		{ "path": "../blob" },
 		{ "path": "../core-data" },
 		{ "path": "../hooks" },
-		{ "path": "../html-entities" }
+		{ "path": "../html-entities" },
+		{ "path": "../block-editor" },
```

Comment:
I think this is a left over (need to be replaced by media-utils maybe)

Reply:
Good catch! Fixed with [6592d36](https://github.com/WordPress/gutenberg/pull/64496/commits/6592d3652ad22d751038fd9bf375ea2da5e9dafa)